manifests/plugin/discovery.pp

```diff
@@ -25,6 +25,12 @@
   if $install_images {
     $tftp_root_clean = regsubst($tftp_root, '/$', '')
 
+    if ! defined(File["${tftp_root_clean}/boot"]) {
```

Comment:
I prefer ```ensure_resource('file', "${tftp_root_clean}/boot", {ensure => directory})```

Reply:
Done. I only wonder why this is now three commits...

Reply:
I think this is still not correct. ;)
Please squash all your commits together and force-push, if possible - however, we can also fix this up on merge...